docs/android/testing/integration_testing.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Android integration testing"
+sidebar_label: "Integration testing"
+---
+
+🚧🚧🚧 Under Construction 🚧🚧🚧
+
+## Why perform integration testing?
+
+[Unit tests](unit_testing) are great and should be your primary choice when writing tests. However, integration testing ensures that the behavior of the application in a real Android environment is validated. Integration tests run on a real Android OS through an emulator, using the same JVM that will be used by end users.
+
+### Testing on a real JVM
+
+Some libraries we use have specific requirements, such as a minimum Java version above 8. Since our application targets Android API 21, which uses Java 8, these libraries cannot always be used. 
```

Comment:
I'm not 100% sure but I believe Android 5 uses Java 7? We need desugaring for `java.time`.

Reply:
Indeed you are right, but it's more complicated than that. It has a partial support for Java 8 https://developer.android.com/studio/write/java8-support that's why it was not clear.